apm-agent-plugins/apm-redis-plugin/apm-jedis-3-tests/src/test/java/co/elastic/apm/agent/redis/jedis/Jedis3InstrumentationTest.java

```diff
@@ -0,0 +1,29 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.redis.jedis;
+
+class Jedis3InstrumentationTest extends Jedis2InstrumentationTest {
```

Comment:
Since it is the same instrumentation and the same API, I believe we are OK with only testing version 3.

Reply:
Version 2 and 3 implement different interfaces. I feel like it's worth the extra testing module, especially because of the way the matcher works for Jedis.